src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import { Todo } from '../../types/Todo';
+import { TodoComponent } from '../TodoComponent';
+
+type Props = {
+  todos: Todo[] | undefined;
```

Comment:
```suggestion
  todos: Todo[];
```

List should be rendered only if todos exist